contrib/grpc/interceptors_test.go

```diff
@@ -0,0 +1,35 @@
+package interceptors
+
+import (
+	"context"
+	"testing"
+
+	"google.golang.org/grpc"
+	"github.com/appoptics/appoptics-api-go"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+	"strings"
+)
+
+var (
+	srv = "server_full_of_testing"
+	uInfo = &grpc.UnaryServerInfo{
+		FullMethod: "/something/blah",
+		Server:     srv,
+	}
+)
+
+func TestUnaryRequest(t *testing.T) {
+	measures := appoptics.NewMeasurementSet()
+	intercept := UnaryServerInterceptor(measures)
+	ctx := context.Background()
+	var handler grpc.UnaryHandler = func(ctx context.Context, req interface{}) (interface{}, error) {
```

Comment:
For a reason that I have yet to identify this variable assignment breaks in go 1.8 and thus the Circle failure. We will have to change this or bump our supported go version up to 1.9 or 1.10 (not sure if it works with 1.9)

Reply:
Thank you for doing that sleuthing :)

I'm going to disable this test until we decide to move to 1.10

Reply:
The answer was type aliasing was added in 1.9, and gRPC for Go uses the x/net/context package rather than the new builtin "context" package for compatibility reasons — upgrading to Go 1.10 makes them the same type.
https://golang.org/doc/go1.9#language